source/renderer/app/components/navigation/NavButton.scss

```diff
@@ -7,6 +7,26 @@
   justify-content: center;
   text-align: center;
   width: 100%;
+
+  &.hasNotification {
+    .container {
+      position: relative;
+      &:after {
+        background: var(--theme-button-attention-background-color);
+        border-radius: 50%;
+        content: '';
+        height: 8px;
+        // position: absolute;
```

Comment:
@daniloprates please check if we need this :pray:

Reply:
✅